GDJS/GDJS/Extensions/Builtin/SceneExtension.cpp

```diff
@@ -39,6 +39,15 @@ SceneExtension::SceneExtension() {
   GetAllConditions()["DoesSceneExist"].SetFunctionName(
       "gdjs.evtTools.runtimeScene.doesSceneExist");
 
+  GetAllActions()["PrioritizeLoadingOfLayout"].SetFunctionName(
```

Comment:
Just realized this. I think we should use Scene instead of `Layout` in the runtime (sorry it's 4 renamings, but I think it's important).